spec/lita/handlers/github_pinger_spec.rb

```diff
@@ -21,4 +22,47 @@
     response = http.post("/ghping", '{"hello": "world", "events": ["pull_request_review_comment"]}')
     expect(response.body).to_not be_nil
   end
+
+  describe 'act on assign' do
+    before do
+      Lita::Room.create_or_update(1, name: 'eng-pr')
+      Lita::User.create(1, name: 'taylorlapeyre')
+    end
+
+    context 'pull request' do
+      it 'sends direct message' do
+        fake_json_request = {
+          action: 'assigned',
+          pull_request: {
+            assignee: {
+              login: 'taylorlapeyre'
+            }
+          }
+        }.to_json
+
+        response = http.post('/ghping', fake_json_request)
+
+        expect(replies.first).to match("You've been assigned to review a pull request")
+        expect(response.status).to eq(200)
+      end
+    end
+
+    context 'issue' do
+      it 'sends direct message' do
+        fake_json_request = {
+          action: 'assigned',
+          issue: {
+            assignee: {
+                login: 'taylorlapeyre'
+            }
+          }
+        }.to_json
+
+        response = http.post('/ghping', fake_json_request)
+
+        expect(replies.first).to match("You've been assigned to review a issue")
+        expect(response.status).to eq(200)
+      end
+    end
+  end
```

Comment:
I was also trying to make an expectation on the Lita User the message was being sent to, but could not figure it out. Settling for this for now.